client/src/modules/reports/generate/configurable_analysis_report/configurable_analysis_report.config.js

```diff
@@ -14,6 +14,7 @@ function ConfigurableAnalysisReportController($sce, Notify, SavedReports, AppCac
   ConfigurationAnalysisTools.read()
     .then((configurationAnalysisTools) => {
       vm.configurationsAreMissing = (configurationAnalysisTools.length === 0);
+      vm.warningStyle = vm.configurationsAreMissing ? 'background-color: #BBB; color: #777;' : '';
```

Comment:
I don't think this is necessary.  Can we just use the regular disabled styles?

Reply:
Decided to hide all form elements if the configuration is missing.